nncf/experimental/onnx/model_normalizer.py

```diff
@@ -19,8 +19,15 @@
 
 
 class ONNXModelNormalizer:
+    """
+    The class, which helps to prepare the ONNX model for work with Post-Training Algorithms.
+    Implements methods for adding necessary information to the ONNX model.
+    """
+    DESIRED_OPSET_VERSION = 13
```

Comment:
`Desired` by whom specifically? Perhaps `TARGET` is a more fitting word?

Reply:
Done